lib/improver/tests/blending/weighted_blend/test_WeightedBlendAcrossWholeDimension.py

```diff
@@ -321,6 +321,7 @@ def test_incompatible_weights_and_data_cubes(self):
         msg = "Weights cube is not a compatible shape with the data cube"
 
         with self.assertRaisesRegex(ValueError, msg):
+
```

Comment:
I enjoy the new found spaciousness provided by this line. I very much felt the feng shui was all wrong before, but now the qi of this test file seems to flow all the better 😄